app/actions/index.js

```diff
@@ -125,7 +168,7 @@ export const clearRemoteSurveys = () => (dispatch, getState) =>
     type: types.CLEAR_REMOTE_SURVEYS
   });
 
-export const fetchRemoteSurvey = (id, url) => (dispatch, getState) => {
+export const fetchRemoteSurvey = (id, url, target) => (dispatch, getState) => {
```

Comment:
We probably only want to sync when fetching the first survey from a peer rather than each time.